google/cloud/pubsub/BUILD

```diff
@@ -40,6 +40,7 @@ cc_library(
         "//google/cloud:google_cloud_cpp_common",
         "//google/cloud:google_cloud_cpp_grpc_utils",
         "@com_google_googleapis//google/pubsub/v1:pubsub_cc_grpc",
+        "@com_google_googletest//:gtest_main",
```

Comment:
It seems strange for a library to depend on `gtest_main`.  Should this just be `gtest`?

Reply:
Fixed